pkg/utils/constants.go

```diff
@@ -90,3 +90,9 @@ const (
 	// Default port used to access vCenter.
 	DefaultVCenterPort string = "443"
 )
+
+const (
+	DataManagerRepository string = "data-manager-for-plugin"
```

Comment:
This is not the repository name, this is the container name.  This is confusing.

Reply:
Changed to DataManagerForVsphere.